bdbstore/src/test/java/org/apache/qpid/server/store/berkeleydb/replication/ReplicatedEnvironmentFacadeTest.java

```diff
@@ -1315,10 +1406,14 @@ private ReplicatedEnvironmentFacade addNode(StateChangeListener stateChangeListe
     }
     private ReplicatedEnvironmentConfiguration createReplicatedEnvironmentConfiguration(String nodeName, String nodeHostPort, boolean designatedPrimary)
     {
-        return createReplicatedEnvironmentConfiguration(nodeName,nodeHostPort,designatedPrimary,false);
+        return createReplicatedEnvironmentConfiguration(nodeName, nodeHostPort, designatedPrimary, false, null);
     }
 
-    private ReplicatedEnvironmentConfiguration createReplicatedEnvironmentConfiguration(String nodeName, String nodeHostPort, boolean designatedPrimary,final boolean disableCoalescing)
+    private ReplicatedEnvironmentConfiguration createReplicatedEnvironmentConfiguration(String nodeName,
```

Comment:
Please revert new durability parameter. I do not think we need to change it in any test...
IMHO, it should be sufficient to set it to NO_SYNC,NO_SYNC,NONE in createReplicatedEnvironmentConfiguration.
It is unlikely that  another durability would be tested.

The changes caused by an introduction of durability parameter can be reverted as well...

Reply:
changed!